qa/restart-upgrade/src/test/java/org/opensearch/knn/bwc/IndexingIT.java

```diff
@@ -0,0 +1,35 @@
+/*
+ *  Copyright OpenSearch Contributors
+ *  SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.knn.bwc;
+
+import static org.opensearch.knn.TestUtils.KNN_BWC_PREFIX;
+
+public class IndexingIT extends AbstractRestartUpgradeTestCase {
+    private final String CLASS_NAME = "indexingit";
+    private final String testIndexName = KNN_BWC_PREFIX + CLASS_NAME + "test-index";
+    private final String testFieldName = "test-field";
+    private final int dimensions = 5;
+    private final int k = 5;
+    private final int addDocsCnt = 10;
+
+    public void testKnnDefaultIndexSettings() throws Exception {
+        waitForClusterHealthGreen("3");
```

Comment:
better use constant for number of nodes. we also can create some default implementation for 3 nodes and pull it to parent abstract class